pkg/gardenlet/controller/seed/seed/reconciler_delete.go

```diff
@@ -200,7 +200,6 @@ func (r *Reconciler) runDeleteSeedFlow(
 		kubeStateMetrics   = kubestatemetrics.New(seedClient, r.GardenNamespace, nil, kubestatemetrics.Values{ClusterType: component.ClusterTypeSeed})
 		nginxIngress       = nginxingress.New(seedClient, r.GardenNamespace, nginxingress.Values{})
 		networkPolicies    = networkpolicies.NewBootstrapper(seedClient, r.GardenNamespace, networkpolicies.GlobalValues{})
-		clusterIdentity    = clusteridentity.NewForSeed(seedClient, r.GardenNamespace, "")
```

Comment:
I assume this was mistakenly added during rebase?

Reply:
It was (re-)moved in https://github.com/gardener/gardener/pull/7436 and is not rebased yet.